### ISSUE TITLE
F-Droid not getting app updates

### ISSUE THREAD

**Issue by esmoak** [Jizzu/SimpleToDo#28]
I am not sure if the best way to ask this is by posting an issue or not, but I was curious why the recent builds of your app haven't made it to F Droid?

**Comment by David-Else** [comment]
Yes, I wanted to download it from there so I can keep up-to-date automatically, but there is only some ancient version.
